### DISCUSSION TITLE
华硕路由器安装失败

### DISCUSSION THREAD

**Discussion by ghostliu1** [AlistGo/alist#2868] [Invalid question]
华硕AX82U路由器，在梅林市场安装alist显示仅支持armv8机型，在项目主页下载3.7.2 armv7版本离线安装，提示没有找到安装脚本install.sh，请问如何解决?
![微信图片_20230102125510](https://user-images.githubusercontent.com/121758510/210196210-634f40ae-af17-4898-8b66-e95317294f01.jpg)
![微信图片_20230102125520](https://user-images.githubusercontent.com/121758510/210196213-dad678f0-eb00-4e1c-9f3b-11d9180b20b8.jpg)
![微信图片_20230102125529](https://user-images.githubusercontent.com/121758510/210196215-b77fa065-8ed6-4f93-8464-b3b9ed0ed1bd.jpg)

**Comment by NovaShen0724** [comment]
发行包不需要安装，故没有这个脚本，解压后为二进制文件，直接运行并配置demon即可
请手动安装

**Reply by NovaShen0724** [reply]
有问题请提交Q&A

**Reply by ghostliu1** [reply]
麻烦详细说说

**Comment by bacontse** [comment]
有没有可能安装错了 https://github.com/messense/aliyundrive-webdav/releases

**Comment by qpwbnu** [comment]
我遇到了和你一样的问题，应该是Alist官方不支持这样安装方式，如果使用ssh安装就没问题了。另外我建议可以尝试华硕固件这边的alist，这个是应用商店里的安装包。https://github.com/everstu/Koolcenter_alist/releases
